packages/invoices/src/OutboundShipment/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -130,7 +130,10 @@ const useStatusChangeButton = () => {
   // When the status of the invoice changes (after an update), set the selected option to the next status.
   // It would be set to the current status, which is now a disabled option.
   useEffect(() => {
-    setSelectedOption(() => getNextStatusOption(status, options));
+    const nextSelectedOption = getNextStatusOption(status, options);
```

Comment:
clumsy attempt to work around this:

![Screen Shot 2022-02-18 at 10 12 17 AM](https://user-images.githubusercontent.com/9192912/154586830-d0af61ed-8254-4490-8eb3-0cf3d13a8465.png)
 
The `setState` was having trouble comparing the selection option against the options, and I didn't see why they were being recreated.

Reply:
Ah yes I just ran up against this. 

turns out `t` from `useTranslation` is a new reference each render, so using it in a deps array causes some unwanted behaviour!

i.e. in here:

```
const options = useMemo(
    () => getStatusOptions(status, getButtonLabel(t)),
    [status, getButtonLabel, t]
  );
```

Taking the `t` out stops the infinite re-rendering (Though, doesn't solve the root cause)

Reply:
aha! didn't notice that .. well spotted